pkg/kubelet/kubelet.go

```
@@ -2967,6 +2992,18 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 			kl.statusManager.SetPodResizeInProgressCondition(pod.UID, "", "", false)
 		} else {
 			// (Allocated == Actual) => clear the resize in-progress status.
+			// If pod resize completed, print ResizeCompleted event
+			resizeStatus := kl.statusManager.GetPodResizeConditions(allocatedPod.UID)
+			ifPodResizeInProgress := slices.ContainsFunc(resizeStatus, func(cond *v1.PodCondition) bool {
+				return cond != nil && cond.Type == v1.PodConditionType("PodResizeInProgress")
+			})
+			if ifPodResizeInProgress {
+				msg, error := getPodResizeInfo(pod)
+				if error == nil {
+					kl.recorder.Eventf(pod, v1.EventTypeNormal, events.ResizeCompleted, msg)
+				}
```

error is ignored, but shouldn't be.